docs/src/modules.md

```diff
@@ -5,12 +5,47 @@
 Currently the easiest way to test Stylix is to use the new code in your
 actual configuration.
 
-You might find it useful to change the flake reference in your configuration
-from `github:danth/stylix` to `git+file:/home/user/path/to/stylix`
-so that you don't need to push your changes to GitHub during testing.
+You might find it useful to override Stylix' input flake reference of your flake
+from `github:danth/stylix` to `git+file:/home/user/path/to/stylix` so that you
+don't need to push changes to GitHub during testing.
 
-Then, remember to run `nix flake lock --update-input stylix` to refresh the
-flake each time you make an edit.
+To do that, instead of editing your `flake.nix`, you can leverage the parameter
+`--override-input` (available to `nix`, `nixos-rebuild`, `nix-on-droid` and
+even `nh`), that allows you to deploy your changes in one fell swoop, without
+having to update the lock file of your flake every time you make an edit.
+
+Just append `--override-input stylix git+file:/home/user/path/to/stylix` to your
+standard incantation to `nix`, `nixos-rebuild` or `nix-on-droid`:
```

Comment:
`nh` is not mentioned here. Maybe there is a neater way of not explicitly listing the specific CLI and just referring to the list that follows this paragraph.